providers/implementations/keymgmt/slh_dsa_kmgmt.c.in

```diff
@@ -339,6 +365,13 @@ static int slh_dsa_fips140_pairwise_test(const SLH_DSA_KEY *key,
                           sig, &sig_len, sig_len) != 1)
         goto err;
 
```

Comment:
OpenSSL uses `OSSL_SELF_TEST_oncorrupt_byte(st, sig);` to corrupt self-tests, so if this function were to do both PCT and KATs, it would have to have at least 3 of those I think: one to corrupt the PCT, and two to corrupt the SigGen and SigVer KATs.

Reply:
Is this a requirement for self-tests?

Reply:
Its a requirement for testing. This sounds like it is getting complicated.